docs/storage.conf.5.md

```diff
@@ -41,9 +40,118 @@ The `storage` table supports the following options:
   container storage driver (default is "overlay")
   Default Copy On Write (COW) container storage driver
 
+##STORAGE TABLE OPTIONS
+
 **additionalimagestores**=[]
   Paths to additional congtainer image stores. Usually these are read/only and stored on remote network shares.
 
+## STORAGE TABLE OPTIONS DEVICEMAPPER
```

Comment:
Unlike on line 29, this doesn't actually tell me what the section should be called.  Can we include that?